docs/Homebrew-Governance.md

```diff
@@ -16,131 +17,132 @@
 
-2. The PLC will appoint between three and five maintainers to be members of the TSC. Voting PLC members should not be any of these appointees. Appointed TSC members will serve a term of one year or until the member's successor is appointed.
+2. The PL is one member of the TSC. The PLC will appoint between three and five maintainers to be members of the TSC. PLC members should not be any of these appointees. Appointed TSC members will serve a term of one year or until the member's successor is appointed.
 
 3. Any member may refer any technical question or dispute to the TSC. Members will make a good faith effort to resolve any disputes with compromise prior to referral to the TSC.
 
 4. No more than two employees of the same employer may serve on the TSC.
 
-5. A member of the TSC, except the Project Leader, may be removed from the TSC by an ordinary resolution of the PLC.
+5. A member of the TSC, except the Project Leader, must only be removed from the TSC by an ordinary resolution of the PLC.
 
 6. All members of the TSC will be "moderators" of the GitHub organisation.
 
-7. One member of the TSC (not the PL) will be an "Owner" of the GitHub organization, Slack, 1Password and any related resources.
+7. One member of the TSC (not the PL) will be an "Owner" of the GitHub organisation, Slack, 1Password and any related resources.
 
 ## 8. Maintainers
 
 1. All maintainers are automatically members. Some, not all, members are maintainers.
 
 2. Maintainers are members with commit/write-access to at least one primary repository.
 
-3. New maintainers can be nominated by any existing maintainer. To become a maintainer, a nomination requires approval from one of the PL or any member of the TSC with no opposition from any of these people within a 24-hour period, excluding 19:00 UTC on Friday until 19:00 UTC on the following Monday. If there is opposition, the TSC must vote on the nomination in the #tsc private Slack channel, with the vote closing after one week or after the outcome of the vote would not be changed by any subsequent votes (such as when a majority of the TSC has voted in favor or against). The nomination will succeed by a simple majority vote of the votes cast.
+3. New maintainers can be nominated by any existing maintainer. To become a maintainer, a nomination requires approval from one of the PL or any member of the TSC with no opposition from any of these people within a 24-hour period, excluding 19:00 UTC on Friday until 19:00 UTC on the following Monday. If there is opposition, the TSC must vote on the nomination in the #tsc private Slack channel, with the vote closing after one week or after the outcome of the vote would not be changed by any subsequent votes (such as when a majority of the TSC has voted in favour or against). The nomination will succeed by a simple majority vote of the votes cast.
```

Comment:
#tsc -> \`#tsc\` (for consistency with \`#members\` on line 65)

Reply:
@maxim-belkin Can you make a GitHub suggestion for this (and any other suggested wording changes)?